larq_compute_engine/mlir/BUILD

```diff
@@ -407,6 +422,7 @@ pybind_extension(
     module_name = "tf_tfl_flatbuffer",
     deps = [
         ":lce_tfl_passes",
+        ":python_common",
         ":tf_to_tfl_flatbuffer",
         "@org_tensorflow//tensorflow/cc/saved_model:loader",
         "@org_tensorflow//tensorflow/compiler/mlir/lite:tensorflow_lite",
```

Comment:
I saw you removed a lot of includes from `graphdef_tfl_flatbuffer.cc` and `saved_model_tfl_flatbuffer.cc`, maybe they could also be removed from the `deps` here? They'll still be used through the `python_common` dependency, but it might be a bit cleaner.

Reply:
Hmm, it is a bit tricky. Technically it works already when I do this:
```bazel
    deps = [
        ":python_common",
    ],
```
But I think despite that I removed a lot of includes, all dependencies listed here are still direct dependencies of `graphdef_tfl_flatbuffer.cc` and `saved_model_tfl_flatbuffer.cc`.

Reply:
Allright. Then it's good to go from my side :+1: